concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -206,13 +218,56 @@ resources:
       key: release-version
       initial_version: 0.0.0
 
+  - name: pipeline-pool
+    type: pool
+    source:
+      uri: {{git_concourse_pool_clone_full_url_ssh}}
+      branch: master
+      pool: {{pipeline_name}}
+      private_key: {{pipeline_lock_git_private_key}}
+
 jobs:
+  - name: pipeline-lock
+    plan:
```

Comment:
I think this job should be [serial](https://concourse.ci/configuring-jobs.html#serial)